lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -214,6 +214,77 @@ speed:
   config_set_append: "speed %s"
   default_value: "auto"
 
+stp_bpdufilter:
+  kind: string
```

Comment:
Ok, that makes sense for `stp_guard`, but `stp_bpdufilter` and `stp_bpduguard` only have two values... `enable` and `disable`.  IMO, these should be treated as booleans IF there cannot be more then these two values.

https://github.com/saichint/cisco-network-puppet-module/blob/feature/stp_global/lib/puppet/type/cisco_interface.rb#L493